docs/guides/export-backup.md

```
@@ -0,0 +1,664 @@
+3. Select **"Export as Markdown"**
+4. File downloads immediately: `document-title.md`
+
+**Markdown Output Includes**:
+- Document title as H1 heading
+- Full formatted content
+- Preserved headings, lists, and formatting
+- Code blocks with syntax markers
+- Tables in markdown format
+- Links preserved
+
+**Example Output**:
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)